src/menus/BenefitPlanMainMenu.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { injectIntl } from "react-intl";
+import { connect } from "react-redux";
+import { Tune } from "@material-ui/icons";
+import { formatMessage, MainMenuContribution, withModulesManager } from "@openimis/fe-core";
+import { SOCIAL_PROTECTION_MAIN_MENU_CONTRIBUTION_KEY } from "../constants";
+
+const BenefitPlanMainMenu = (props) => {
+    const entries = [
+        {
+            text: formatMessage(props.intl, "socialProtection", "menu.socialProtection.benefitPlans"),
+            icon: <Tune />,
+            route: "/benefitPlans",
+        }
+    ];
+    entries.push(
+        ...props.modulesManager
+        .getContribs(SOCIAL_PROTECTION_MAIN_MENU_CONTRIBUTION_KEY)
+            .filter((c) => !c.filter || c.filter(props.rights)),
+    );
+
+    return (
+        <MainMenuContribution {...props} header={formatMessage(props.intl, "socialProtection", "mainMenuSocialProtection")} entries={entries} />
```

Comment:
I wonder if we should put such benefit plan under 'Beneficiary & Household' menu instead of creating separate menu point. 
That comment is only a proposition of improvement.

Reply:
@dborowiecki What do you think? What should be the menu structure?

Reply:
It'll be latter on adjusted im mocks, for now let's keep it as it is

Reply:
ok

Reply:
then I think I can resolve this conversation.